types/apimocker/index.d.ts

```diff
@@ -0,0 +1,49 @@
+// Type definitions for apimocker 1.1
+// Project: https://www.npmjs.com/package/apimocker
+// Definitions by: Uchenna <https://github.com/uchilaka>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.5
```

Comment:
This should be the lowest possible version in which these definitions work. I don’t see any bleeding-edge new TypeScript features in here, so you should be safe to use whatever `express` uses—[2.2](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/c79180dd81e7e726ef7763320ddbdc099eaec995/types/express/index.d.ts#L5). That will let people stuck on older TypeScript versions still benefit from these typings 👍